### PR TITLE
Openldap: Creation

### PR DESCRIPTION
**Pull request by ekozan**
WIP
For work tracking
Please dont test it ;) 

Todo:

- Add more instruction
- Add reference
- Make test

### CODE REVIEW

**Comment by viglesiasce** [comment]
@k8s-bot ok to test

File: incubator/openldap/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Openldap
+
+__DISCLAIMER:__ While this chart has been well-tested, testers have encountered occasional issues
```
Comment (viglesiasce) [inline incubator/openldap/README.md:3]: This looks like a copy paste issue.

File: incubator/openldap/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+with the Traefik software itself. Be advised that your mileage may vary.
+
+## Introduction
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+- You are deploying the chart to a cluster with a cloud provider capable of provisioning an
```
Comment (viglesiasce) [inline incubator/openldap/README.md:11]: Is this true?

**Review by viglesiasce — CHANGES_REQUESTED** [review]
Please fix the readme and add NOTES.txt

File: incubator/openldap/README.md (hunk 1 above)
Comment (ekozan) [reply incubator/openldap/README.md:3]: spoted

File: incubator/openldap/README.md (hunk 2 above)
Comment (ekozan) [reply incubator/openldap/README.md:11]: spoted v2

**Comment by mgoodness** [comment]
@ekozan is this still an active/WIP PR? Do you have an ETA for completion?

**Comment by mgoodness** [comment]
Closing due to inactivity. Feel free to re-open if/when ready.

**Pull request closed without merging**